.lintstagedrc

```diff
@@ -1,3 +1,3 @@
 {
-    "*.(js|ts|tsx|jsx)": ["prettier --write", "eslint --cache --fix", "pnpm test -- --bail --findRelatedTests"]
+    "*.(js|ts|tsx|jsx)": ["prettier --write", "eslint --cache --fix", "pnpm test -- --bail --findRelatedTests --passWithNoTests"]
```

Comment:
By the way, as we touched this file, could you remove running tests on the commit?

Reply:
Well, I like running tests on a commit :) Is there an issue with that?

Reply:
We can move test running to prepush hook.

Reply:
@dangreen agree. 

@tenphi Running tests on commit slow down your work. If you want to reduce the feedback loop, run tests while you develop a feature using `--watch` flag.

IMHO, preventing committing "inappropriate" code is a bad practice that slows development and gives nothing in return.

What if you must commit current state of work, which might not be "perfect"? There might be plenty of reasons for that.